### PR TITLE
Included docs for POST,PUT,DELETE for v3,v4,v5 of statuses and statusesID

### PR DESCRIPTION
**Pull request by heneriknguyen**
Closes: #6318 

<hr/>

## Which Traffic Control components are affected by this PR?

- Documentation

## What is the best way to verify this PR?

Ensure the docs are working correctly.

## If this is a bugfix, which Traffic Control versions contained the bug?
7.0.1

## PR submission checklist
- [ ] This PR has tests 
- [x] This PR has documentation 
- [x] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

File: docs/source/api/v4/statuses_id.rst [hunk 1]
```diff
@@ -156,3 +156,62 @@ Response Structure
...
+:Response Type:  ``undefined``
+
+Request Structure
+-----------------
+.. table:: Request Path Parameters
+
+	+------+----------+--------------------------------------------------------------------------------------------------------------------------+
+	| Name | Required | Description                                                                                                              |
+	+======+==========+==========================================================================================================================+
+	| id   | yes      | The integral, unique identifier of the desired :abbr:`Status`-to-:term:`Server` association |
```
Comment (ocket8888) [inline docs/source/api/v4/statuses_id.rst:176]: Malformed table

File: docs/source/api/v5/statuses_id.rst [hunk 2]
```diff
@@ -156,3 +156,60 @@ Response Structure
...
+:Response Type:  ``undefined``
+
+Request Structure
+-----------------
+.. table:: Request Path Parameters
+
+	+------+----------+--------------------------------------------------------------------------------------------------------------------------+
+	| Name | Required | Description                                                                                                              |
+	+======+==========+==========================================================================================================================+
+	| id   | yes      | The integral, unique identifier of the desired :abbr:`Status`-to-:term:`Server` association |
```
Comment (ocket8888) [inline docs/source/api/v5/statuses_id.rst:176]: Malformed table

**Review by ocket8888 — CHANGES_REQUESTED** [review]

**Comment by jagan-parthiban** [comment]
In this PR, for V4 and V5, this is the docs structure added by you,
```
statuses
- GET
- POST
- PUT
- DELETE

statuses/{{ID}}
- GET
- PUT
- DELETE
```

I think we have to remove `PUT` and `DELETE` from `statuses` and remove `GET` from` statuses/{{ID}}`. It should be like any other documentation,
```
statuses
- GET
- POST

statuses/{{ID}}
- PUT
- DELETE
```

References: 
[divisions](https://traffic-control-cdn.readthedocs.io/en/latest/api/v5/divisions.html)
[divisions/{{ID}}](https://traffic-control-cdn.readthedocs.io/en/latest/api/v5/divisions_id.html)

**Review by jagan-parthiban — APPROVED** [review]

File: docs/source/api/v5/statuses.rst [hunk 3]
```diff
@@ -156,3 +156,4 @@ Response Structure
 			"lastUpdated": "2018-12-10 19:11:17+00",
 		}
 	]}
+
```
Comment (rimashah25) [inline docs/source/api/v5/statuses.rst:159]: added a new line by mistake.

**Review by rimashah25 — APPROVED** [review]
LGTM

File: docs/source/api/v5/statuses_id.rst [hunk 4]
```diff
@@ -156,3 +156,60 @@ Response Structure
 			"lastUpdated": "2018-12-10 19:11:17+00",
 		}
 	]}
+
```
Comment (rimashah25) [inline docs/source/api/v5/statuses_id.rst:159]: DELETE should be part of statuses_id

**Review by rimashah25 — CHANGES_REQUESTED** [review]

**Pull request merged**